code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -518,14 +518,31 @@
 
 	QDEL_LIST(temp_objs)
 
-	// Chance for souvenir or bruises
+	// Injuries due to questioning and souvenirs
 	if(prob(RETURN_SOUVENIR_CHANCE))
 		to_chat(M, "<span class='notice'>Your captors left you a souvenir for your troubles!</span>")
 		var/obj/item/souvenir = pick(souvenirs)
 		new souvenir(closet)
-	else if(prob(RETURN_BRUISE_CHANCE) && M.health >= 50)
-		to_chat(M, "<span class='warning'>You were roughed up a little by your captors before being sent back!</span>")
-		M.adjustBruteLoss(RETURN_BRUISE_DAMAGE)
+	if(prob(RETURN_INJURY_CHANCE) && M.health >= 50)
+		var/obj/item/organ/external/injury_target
+		to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back!</span>")
+		if(prob(20)) //remove a limb
+			if(prob(50))
+				injury_target = M.get_organ(BODY_ZONE_R_ARM)
```

Comment:
Why just the right arm in specific, why not pick between right and left arm?

Reply:
I flipped a coin, honestly. I want to see if there’s a nice way to fully randomize it between your extremities but I have to figure out a non-scuffed way of doing it.

Reply:
Today we learn of a proc, that works perfectly.